projects/security.md

```diff
@@ -0,0 +1,82 @@
+  * This includes but is not limited to the following namespaces:
+    * [`Code signature`](https://www.elastic.co/guide/en/ecs/current/ecs-code_signature.html)
+    * [`DLL`](https://www.elastic.co/guide/en/ecs/current/ecs-dll.html)
+    * [`DNS`](https://www.elastic.co/guide/en/ecs/current/ecs-dns.html)
+    * [`File`](https://www.elastic.co/guide/en/ecs/current/ecs-file.html)
+    * [`Group`](https://www.elastic.co/guide/en/ecs/current/ecs-group.html)
+    * [`Hash`](https://www.elastic.co/guide/en/ecs/current/ecs-hash.html)
+    * [`Host`](https://www.elastic.co/guide/en/ecs/current/ecs-host.html)
+    * [`Network`](https://www.elastic.co/guide/en/ecs/current/ecs-network.html)
+    * [`Operating System`](https://www.elastic.co/guide/en/ecs/current/ecs-os.html)
+    * [`Package`](https://www.elastic.co/guide/en/ecs/current/ecs-package.html)
+    * [`Process`](https://www.elastic.co/guide/en/ecs/current/ecs-process.html)
+    * [`Registry`](https://www.elastic.co/guide/en/ecs/current/ecs-registry.html)
+    * [`Risk information`](https://www.elastic.co/guide/en/ecs/current/ecs-risk.html)
+    * [`Rule`](https://www.elastic.co/guide/en/ecs/current/ecs-rule.html)
+    * [`Threat`](https://www.elastic.co/guide/en/ecs/current/ecs-threat.html)
+    * [`TLS`](https://www.elastic.co/guide/en/ecs/current/ecs-tls.html)
+    * [`User`](https://www.elastic.co/guide/en/ecs/current/ecs-user.html)
+    * [`Vulnerability`](https://www.elastic.co/guide/en/ecs/current/ecs-vulnerability.html)
+  * Please note that some of the above-mentioned namespaces are already a part of the Semantic Conventions schema. The goal is to expand these namespaces to include additional fields that are relevant to security use cases.
+
+* As new use cases and namespaces are introduced to the semantic conventions, there may be a need for additional instrumentation to accommodate them. It is anticipated that this aspect will expand through an iterative process.
+
+## Staffing / Help Wanted
+
+We are seeking security experts to collaborate with us in expanding the security domain within the community.  
+
+### Required staffing
+
+There is an open [PR](https://github.com/open-telemetry/semantic-conventions/issues/580) to create a `semconv-security-approver` group for all PRs related to security fields.  
+
+* project lead: @trisch-me (Elastic)
+* domain expert: @mjwolf (Elastic)
+* domain expert: @raesene (Datadog)
+* domain expert: @lambdanis (Isovalent)
+* domain expert: @mdelfabro (Dynatrace)
+
+* TC sponsor: @reyang
+* TC sponsor: @jsuereth
+
+Need more 
+- [ ] domain experts
+- [ ] TC 
+- [ ] potentially, maintainers of language-specific instrumentation may be needed if the need arises.
+
+
+## Meeting Times
+
+There is an allocated time in the Semantic Conventions WG for this project.
+- Mondays at 8 AM PST
+
+For async conversation please use #otel-security-wg slack channel from official CNCF slack workspace.
```

Comment:
to help differentiate with #otel-sig-security (and related to #2044)

```suggestion
For async conversation please use #otel-semconv-security slack channel from official CNCF slack workspace.
```

Reply:
While currently we are focusing on introducing semconv as a first milestone, I hope we will use this group also for future updates to the open telemetry such as instrumentation as well. 
Saying that I'm ok to change the name of the group for now. 

wdyt?

Reply:
> I hope we will use this group also for future updates to the open telemetry such as instrumentation as well.

I think it's a good thing for semconv groups to also implement instrumentation which follows the semconv 👍